### PR TITLE
Add force (Single)

### PR DESCRIPTION
**Pull request by EmilRosenquist**
### Add possibility to stream force (single) to unreal
image

* Add option for Force
* Default `false`
* Live link force uses basic role (no predefined static/frame values)
Each force plate is an live link subject and contains the following 9 properties
image

### CODE REVIEW

File: Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Public/QTMConnectLiveLinkSource.h [hunk 1]
```diff
@@ -12,14 +12,16 @@
 #include "SocketSubsystem.h"
 #include <memory>
 #include <vector>
+#include <unordered_map>
+#include <utility>
```
Comment (qjgn) [inline Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Public/QTMConnectLiveLinkSource.h]: Is utility needed?

File: Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLinkEditor/Private/QTMConnectLiveLinkSourceEditor.h [hunk 2]
```diff
@@ -64,6 +64,11 @@ class QTMCONNECTLIVELINKEDITOR_API SQTMConnectLiveLinkSourceEditor : public SCom
         return IsStreamSkeleton();
     }
 
+    bool GetStreamForce() const
+    {
+        return IsStreamForce();
+    }
+
```
Comment (qjgn) [inline Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLinkEditor/Private/QTMConnectLiveLinkSourceEditor.h:71]: Can we remove IsMethods and only have GetMethods?

**Review by qjgn — APPROVED** [review]

File: Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Public/QTMConnectLiveLinkSource.h (hunk 1 above)
Comment (EmilRosenquist) [reply Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Public/QTMConnectLiveLinkSource.h]: Ye nice catch, Removed it

File: Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLinkEditor/Private/QTMConnectLiveLinkSourceEditor.h (hunk 2 above)
Comment (EmilRosenquist) [reply Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLinkEditor/Private/QTMConnectLiveLinkSourceEditor.h:71]: Ye there is some stuff that could be cleaned up here but leaving that for future PR.

**Pull request merged**